src/lib.rs

```diff
@@ -383,13 +400,27 @@ impl TcpKeepalive {
     #[cfg(all(
         feature = "all",
         any(
+            doc,
```

Comment:
Is this supposed to be here?

Reply:
Yes, its purpose is to have this function be documented even on platforms that do not support it (to reduce the need for users of the library to view the docs on multiple platforms).